src/samples/fileMenu/src/stores/fileMenuStore.js

```diff
@@ -156,7 +156,11 @@ var Actions = {
 		FSBL.Clients.LauncherClient.spawn("About Finsemble");
 	},
 	spawnDocs() {
-		FSBL.Clients.LauncherClient.spawn("Finsemble Documentation");
+		fin.desktop.System.openUrlWithBrowser("https://documentation.chartiq.com/finsemble/tutorial-gettingStarted.html", function () {
+			console.log("successfully launched docs");
+		},function (err) {
+			console.log("failed to launch docs");
```

Comment:
Should be console.error.

Should also write the actual error with the summary message:
```js
console.error(`Failed to launch documentation: ${err}`):
```